File: 프로그래머스/0/181908. 접미사인지 확인하기/접미사인지 확인하기.cpp
#include <string>
#include <vector>

using namespace std;

int solution(string my_string, string is_suffix) {
    int answer = 0;
    for(int i = 0; i < my_string.size(); i++)
    {
        string temp = my_string.substr(my_string.size() - 1 - i);
        if(temp == is_suffix)
        {
            answer = 1;
            break;
        }
    }
    return answer;
}

File: 프로그래머스/0/181908. 접미사인지 확인하기/접미사인지 확인하기.cs
using System;

public class Solution {
    public int solution(string my_string, string is_suffix) {
        int answer = 0;
        for(int i = 0 ; i < my_string.Length; i++)
        {
            string str = my_string.Substring(i,my_string.Length-i);
            if(str == is_suffix)
            {
                answer = 1;
                break;
            }
        }
        return answer;
    }
}